spyder/utils/pixi.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Conda/anaconda utilities."""
+
+# Standard library imports
+from functools import lru_cache
+import os
+
+# Third-party imports
+from packaging.version import parse
+
+# Local imports
+from spyder.utils.programs import find_program, run_program
+
+WINDOWS = os.name == "nt"
+
+
+def find_pixi(pyexec=None):
+    """
+    Find pixi executable.
+
+    `pyexec` is a python executable, the relative location from which to
+    attempt to locate a pixi executable.
+    """
+    # Try the environment variables
+    pixi_home = os.environ.get('PIXI_HOME')
+    pixi = None
+    if pixi_home:
+        pixi = os.environ.get('PIXI_HOME') or os.environ.get('MAMBA_EXE')
+
+    # Next try searching for the executable in default paths
+    if pixi is None:
+        pixi_exec = 'pixi.exe' if WINDOWS else 'pixi'
+        pixi = find_program(pixi_exec)
+
+    return pixi
+
+
+@lru_cache(maxsize=1)
+def conda_version(pixi_executable=None):
```

Comment:
This is almost identical to `conda_version` in `spyder/utils/conda.py`. So, I propose to rename that function to `conda_like_version` in that module and its kwarg to `conda_like_executable`.

Then, we wouldn't need this one.

Reply:
No, if you don't deem it necessary. You can just extend its docstring to mention that it gets the version of conda, mamba, micromamba and pip.

But thinking about it, you could also try to add the functionality of `find_pixi` to `find_conda`, if you see it's simple to do.